### PR TITLE
Gemfile updates

### PR DESCRIPTION
**Pull request by ollyjshaw**
Clean up the gemfile. Also the bundle update should not be done at runtime for two reasons:
1.It start up process as docker can't cache it.
2.It could bring in Gem updates that are unexpected and could break thigns

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -26,9 +26,6 @@ gem 'oily_png', '~> 1.1.1'
 gem 'nokogiri', '~> 1.5.10'
 gem 'therubyracer', platforms: :ruby, require: linux_only('therubyracer')
 gem 'aweplug', github: 'awestruct/aweplug'
-#gem 'aweplug', github: 'Dantheman720/aweplug', :branch => 'DEVELOPER-1248'
-#gem 'aweplug', path: '~/projects/ruby/aweplug'
-#gem 'aweplug', path: '../aweplug'
```
Comment (paulrobinson) [inline Gemfile]: These are useful during development, for when we need to point to a different Aweplug location. We have them commented out for convenience.

Comment (LightGuard) [inline Gemfile]: +1 for leaving these in

**Comment by LightGuard** [comment]
If the ones commented out that are mentioned on the comments stay, I'm good with this PR

File: Gemfile [hunk 2]
```diff
@@ -57,7 +52,5 @@ end
 
 group :health do
   gem 'blinkr', '~> 0.2.7'
-  #gem 'blinkr', github: 'pmuir/blinkr'
-  #gem 'blinkr', path: '../blinkr'
```
Comment (paulrobinson) [inline Gemfile]: Looks like these are useful too. @LightGuard?

File: Gemfile [hunk 3]
```diff
@@ -47,8 +44,6 @@ group :development do
   gem 'rake', '~> 10.0.4'
   gem 'pry', require: false
   gem 'pry-byebug', require: false
-  #gem 'pry-stack_explorer', require: false
-  #gem 'pry-exception_explorer', require: false
```
Comment (paulrobinson) [inline Gemfile]: @LightGuard  Useful during debugging?

File: Gemfile (hunk 2 above)
Comment (LightGuard) [inline Gemfile]: Eh, not really. I wouldn't expect anyone else to use a development version
of blinkr

On Wednesday, August 5, 2015, Paul Robinson <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/redhat-developer/developers.redhat.com/pull/348#discussion_r36330808>
> :
>
> > @@ -57,7 +52,5 @@ end
> >
> >  group :health do
> >    gem 'blinkr', '~> 0.2.7'
> > -  #gem 'blinkr', github: 'pmuir/blinkr'
> > -  #gem 'blinkr', path: '../blinkr'
>
> Looks like these are useful too. @LightGuard
> <https://github.com/LightGuard>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/348/files#r36330808>
> .
>


-- 
Sent from Gmail Mobile

File: Gemfile (hunk 3 above)
Comment (LightGuard) [inline Gemfile]: No, not really.

On Wednesday, August 5, 2015, Paul Robinson <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/redhat-developer/developers.redhat.com/pull/348#discussion_r36330854>
> :
>
> > @@ -47,8 +44,6 @@ group :development do
> >    gem 'rake', '~> 10.0.4'
> >    gem 'pry', require: false
> >    gem 'pry-byebug', require: false
> > -  #gem 'pry-stack_explorer', require: false
> > -  #gem 'pry-exception_explorer', require: false
>
> @LightGuard <https://github.com/LightGuard> Useful during debugging?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/348/files#r36330854>
> .
>


-- 
Sent from Gmail Mobile

File: Gemfile (hunk 1 above)
Comment (ollyjshaw) [inline Gemfile]: I'll put them back, I still feel they are noisy though.

**Pull request merged**